editor/src/clj/editor/engine_profiler.clj

```diff
@@ -51,14 +51,22 @@
                     "text/html")]
     mime-type))
 
-(defn- read-resource [req]
+(defn- try-local-path [req]
   (let [path (str "." (:url req)) ; Make it a relative path as it starts with a "/"
-        mime-type (get-mime-type path)
-        resource (io/resource path)
-        data (if (= resource nil)
-               nil
-               (slurp resource))
-        code (if (= data nil) 404 200)]
+        resource (io/resource path)]
+    resource))
+
+(defn- try-package-path [req]
+  (let [path (subs (:url req) 1) ; remove the initial "/"
+        resource (io/resource path)]
+    resource))
+
+(defn- read-resource [req] 
+  (let [mime-type (get-mime-type (:url req))
+        resource (try-package-path req)
+        resource (if (nil? resource) (try-local-path req) resource)
+        data (if (nil? resource) nil (slurp resource))
+        code (if (nil? data) 404 200)]
```

Comment:
Minor: IMO easier to understand. 
```suggestion
        code (if data 200 404)]
```

Reply:
Yeah, not sure why I didn't get it to work the first time. It works now 🤷